blueocean-core-js/src/js/utils/UrlUtils.js

```diff
@@ -1,4 +1,13 @@
 import AppConfig from '../config';
+
+function jobPrefixPath(organization) {
+    // custom organization
+    if (organization !== 'undefined' && organization !== 'jenkins') {
```

Comment:
is the value of the string ever really "undefined"? That seems a little odd to me. Perhaps you could elaborate on this?